lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -49,58 +52,88 @@ protected function configure()
     <info>%command.full_name% --editor-cmd=mate</info>
 EOT
             )
-            ->addOption('filter-expression', null, InputOption::VALUE_OPTIONAL, 'Tables which are filtered by Regular Expression.')
-            ->addOption('formatted', null, InputOption::VALUE_NONE, 'Format the generated SQL.')
-            ->addOption('line-length', null, InputOption::VALUE_OPTIONAL, 'Max line length of unformatted lines.', 120)
+            ->addOption(
+                'filter-expression',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Tables which are filtered by Regular Expression.'
+            )
+            ->addOption(
+                'formatted',
+                null,
+                InputOption::VALUE_NONE,
+                'Format the generated SQL.'
+            )
+            ->addOption(
+                'line-length',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Max line length of unformatted lines.',
+                120
+            )
         ;
     }
 
-    public function execute(InputInterface $input, OutputInterface $output)
-    {
+    public function execute(
+        InputInterface $input,
+        OutputInterface $output
+    ) : void {
         $isDbalOld     = (DbalVersion::compare('2.2.0') > 0);
         $configuration = $this->getMigrationConfiguration($input, $output);
 
         $this->loadCustomTemplate($configuration, $output);
 
-        $conn     = $configuration->getConnection();
+        $conn = $configuration->getConnection();
+
         $platform = $conn->getDatabasePlatform();
 
-        if ($filterExpr = $input->getOption('filter-expression')) {
+        $filterExpr = $input->getOption('filter-expression');
+
+        if ($filterExpr) {
             if ($isDbalOld) {
```

Comment:
This whole thing needs to be dropped - migrations require ^2.6.

Reply:
Recorded issue for this in #625